_sass/components/wells.scss

```diff
@@ -91,3 +91,34 @@
     }
   }
 }
+
+.well-in-development {
+  align-items: center;
+  display: inline-flex;
+  font-size: 15px;
+  margin-bottom: $md;
+  padding: $xs $sm;
+
+  @include media(sm) {
+    width: 100%;
+  }
+
+  @include media(xxs) {
+    align-items: flex-start;
+    flex-direction: column;
+  }
```

Comment:
Updated with center-alignment. Decided not to do the negative margin thing though. I think this element looks okay as-is, but at some point I think we could revisit our overall responsive margins at this size.